Assignment7/src/main/java/com/assignment/springmaven/controllers/EmployeeController.java

```diff
@@ -0,0 +1,53 @@
+package com.assignment.springmaven.controllers;
+
+import com.assignment.springmaven.model.Employee;
+import com.assignment.springmaven.service.EmployeeServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+import java.util.List;
+
+@Controller
+public class EmployeeController {
+
+    @Autowired
+    private EmployeeServiceImpl employeeService;
+
+    @RequestMapping(value = "/addEmployee")
+    public ModelAndView showForm() {
+        return new ModelAndView("/addEmployee","command",new Employee());
+    }
+
+    @RequestMapping(value="/save",method = RequestMethod.POST)
+    public ModelAndView save(@ModelAttribute("employee") Employee employee) throws Exception {
+        employeeService.save(employee);
+        return new ModelAndView("redirect:/viewEmployees");
+    }
+
+    @RequestMapping(value = "/viewEmployees")
+    public ModelAndView viewEmployees() throws Exception {
+        List<Employee> list=employeeService.getEmployees();
+        ModelAndView modelAndView = new ModelAndView("/viewEmployees");
+        modelAndView.addObject("list",list);
+        return modelAndView;
+    }
+
+    @RequestMapping(value="/editEmployee/{id}")
```

Comment:
Mapping should be properly related to the function, you can use "/employee/{id}" to fetch employee details